portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,31 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  CommentList commentList = new CommentList();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello stranger! Welcome to my portfolio");
+    // Convert object to JSON
```

Comment:
This may just be carried over from a previous pull and will probably get updated by your previous PR, but make sure to include periods at the end of implementation comments (here and throughout the file).

Reply:
Done!